### ISSUE TITLE
Annotation markers not appearing in righthand sidebar

### ISSUE THREAD

**Issue by cmbirk** [opengovfoundation/madison#701]
From Fraser.  

I'm seeing this error in the console:

```
TypeError: Cannot read property 'top' of undefined
    at loadAnnotations (app.js:217)
    at app.js:175
    at h.$broadcast (angular.js:110)
    at broadcastUpdate (app.js:3833)
    at setAnnotations (app.js:3740)
    at HTMLElement.<anonymous> (annotator.js:127)
    at HTMLElement.closure (annotator.js:65)
    at HTMLElement.n.event.dispatch (components.js:3)
    at HTMLElement.r.handle (components.js:3)
    at Object.n.event.trigger (components.js:3)(anonymous function) @ angular.js:87
```

**Comment by krues8dr** [comment]
dupe of #548 I think.

**Comment by cmbirk** [comment]
Yeah, possibly.  I'm not seeing the detached annotations though, it looks like they just aren't getting added.

**Comment by krues8dr** [comment]
That'd happen if the previous document didn't have *any* annotations.  ;)

**Comment by krues8dr** [comment]
Either way, the error is the same.

**Comment by cmbirk** [comment]
This is fixed in 08501d0, but still need to deal with the other annotation weirdness moving between documents
